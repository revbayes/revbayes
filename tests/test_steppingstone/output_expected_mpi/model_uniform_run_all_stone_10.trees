Iteration	Posterior	Likelihood	Prior	psi
0	-27740.13	-27724.96	-15.16815	(Galeopterus_variegatus[&index=23]:0.310758,(Lepilemur_hubbardorum[&index=11]:0.310848,Galago_senegalensis[&index=8]:0.231329)[&index=35]:0.056562,(Aotus_trivirgatus[&index=1]:0.054648,(((Otolemur_crassicaudatus[&index=16]:0.197218,Cebus_albifrons[&index=3]:0.053041)[&index=43]:0.078526,Saimiri_sciureus[&index=20]:0.124875)[&index=26]:0.019310,((((Macaca_mulatta[&index=13]:0.134590,((Propithecus_coquereli[&index=19]:0.150591,((Chlorocebus_aethiops[&index=5]:0.078184,Varecia_variegata_variegata[&index=22]:0.170659)[&index=38]:0.169461,Lemur_catta[&index=10]:0.238264)[&index=31]:0.232336)[&index=27]:0.122087,Loris_tardigradus[&index=12]:0.341106)[&index=29]:0.016852)[&index=28]:0.094381,Daubentonia_madagascariensis[&index=7]:0.434275)[&index=40]:0.041124,Cheirogaleus_major[&index=4]:0.115319)[&index=33]:0.170083,(Colobus_guereza[&index=6]:0.145179,(((Callicebus_donacophilus[&index=2]:0.116111,(Tarsius_syrichta[&index=21]:0.139853,Pan_paniscus[&index=17]:0.309392)[&index=36]:0.073783)[&index=42]:0.132662,((Perodicticus_potto[&index=18]:0.209748,Microcebus_murinus[&index=14]:0.040215)[&index=41]:0.038243,Hylobates_lar[&index=9]:0.138412)[&index=39]:0.016413)[&index=32]:0.217618,Nycticebus_coucang[&index=15]:0.062941)[&index=25]:0.027557)[&index=24]:0.128369)[&index=34]:0.001309)[&index=30]:0.236987)[&index=37]:0.219181)[&index=44];
10	-30002.76	-29999.21	-3.551509	(Galeopterus_variegatus[&index=23]:0.232202,((Hylobates_lar[&index=9]:0.295273,Colobus_guereza[&index=6]:0.110750)[&index=41]:0.041576,(((Macaca_mulatta[&index=13]:0.047679,(((((Cebus_albifrons[&index=3]:0.034530,(Callicebus_donacophilus[&index=2]:0.086686,Aotus_trivirgatus[&index=1]:0.147241)[&index=25]:0.085582)[&index=37]:0.085280,Otolemur_crassicaudatus[&index=16]:0.173349)[&index=38]:0.193752,Lemur_catta[&index=10]:0.044452)[&index=26]:0.056672,Tarsius_syrichta[&index=21]:0.100511)[&index=43]:0.078547,(Pan_paniscus[&index=17]:0.107361,Loris_tardigradus[&index=12]:0.140175)[&index=36]:0.319018)[&index=33]:0.077442)[&index=40]:0.054224,(((Varecia_variegata_variegata[&index=22]:0.359073,Propithecus_coquereli[&index=19]:0.194661)[&index=27]:0.027485,Chlorocebus_aethiops[&index=5]:0.056713)[&index=29]:0.013497,Cheirogaleus_major[&index=4]:0.075372)[&index=28]:0.046361)[&index=31]:0.086016,((((Galago_senegalensis[&index=8]:0.114958,Saimiri_sciureus[&index=20]:0.223262)[&index=35]:0.094706,Daubentonia_madagascariensis[&index=7]:0.124066)[&index=30]:0.093307,Lepilemur_hubbardorum[&index=11]:0.220181)[&index=42]:0.114759,(Perodicticus_potto[&index=18]:0.092467,Microcebus_murinus[&index=14]:0.161768)[&index=39]:0.005164)[&index=32]:0.091600)[&index=34]:0.002090)[&index=24]:0.126741,Nycticebus_coucang[&index=15]:0.134853)[&index=44];
