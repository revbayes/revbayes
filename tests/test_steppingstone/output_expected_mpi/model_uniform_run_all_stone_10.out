state	power	likelihood
4	1.16658e-302	-30084.7
5	1.16658e-302	-28407.5
6	1.16658e-302	-28767.2
7	1.16658e-302	-29279.2
8	1.16658e-302	-27847.9
9	1.16658e-302	-28063.7
10	1.16658e-302	-29999.2
